src/server/cmd/worker/main.go

```diff
@@ -46,18 +47,16 @@ func getPipelineInfo(pachClient *client.APIClient, env serviceenv.ServiceEnv) (*
 	defer cancel()
 	pipelines := ppsdb.Pipelines(env.GetDBClient(), env.GetPostgresListener())
 	pipelineInfo := &pps.PipelineInfo{}
-	if err := pipelines.ReadOnly(ctx).Get(env.Config().PPSPipelineName, pipelineInfo); err != nil {
-		return nil, err
-	}
-	pachClient.SetAuthToken(pipelineInfo.AuthToken)
-	// Notice we use the SpecCommitID from our env, not from etcd. This is
-	// because the value in etcd might get updated while the worker pod is
+	// Notice we use the SpecCommitID from our env, not from postgres. This is
+	// because the value in postgres might get updated while the worker pod is
 	// being created and we don't want to run the transform of one version of
 	// the pipeline in the image of a different verison.
-	pipelineInfo.SpecCommit.ID = env.Config().PPSSpecCommitID
```

Comment:
did you mean to keep this line's functionality? The comment suggests that it should still be here

Reply:
Ah, nice catch. I did, but the condition looks different now. The `Sprintf` I didn't hook up should be the collection key that's used